sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -5967,6 +5998,18 @@ export class BlobLeaseClient {
       "BlobLeaseClient-acquireLease",
       options.tracingOptions
     );
+
+    if (
+      this._isContainer &&
+      ((options.conditions?.ifMatch && options.conditions?.ifMatch !== ETagNone) ||
+        (options.conditions?.ifNoneMatch && options.conditions?.ifNoneMatch !== ETagNone) ||
+        options.conditions?.ifTags)
+    ) {
+      throw new RangeError(
+        "the IfMatch, IfNoneMatch and IfTags access conditions must have their default values because they are ignored by the service"
```

Comment:
>the IfMatch, IfNoneMatch and IfTags access conditions must have their default values because they are ignored by the service [](start = 9, length = 124)

What about "the IfMatch, IfNoneMatch and IfTags access conditions are ignored by the service"

Reply:
Decided to go with "The IfMatch, IfNoneMatch and IfTags access conditions are ignored by the service. Values other than undefined or their default values are not acceptable."